src/ansiblelint/config.py

```diff
@@ -23,7 +23,7 @@
 
 
 CACHE_DIR = (
-    os.path.expanduser(os.environ.get("XDG_CONFIG_CACHE", "~/.cache")) + "/ansible-lint"
+    os.path.expanduser(os.environ.get("XDG_CONFIG_HOME", "~/.cache")) + "/ansible-lint"
```

Comment:
What about $XDG_CACHE_HOME ?
Ref: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html:
"$XDG_CACHE_HOME defines the base directory relative to which user-specific non-essential data files should be stored. If $XDG_CACHE_HOME is either not set or empty, a default equal to $HOME/.cache should be used."

Reply:
Yeah. I had it right in the first commit subject but managed to mess up all the rest.
Should be XDG_CACHE_HOME everywhere now.